config/bundle-config.js

```diff
@@ -34,6 +34,15 @@ const DEFAULT_DATA_SOURCES = exports.DEFAULT_DATA_SOURCES = [
       'neuroglancer/async_computation/decode_gzip',
     ],
   },
+  {
+    source: 'neuroglancer/datasource/deepzoom',
+    asyncComputation: [
+      'neuroglancer/async_computation/decode_jpeg',
+      'neuroglancer/async_computation/decode_gzip',
```

Comment:
I think gzip can also be removed, shouldn't be relevant to deepzoom.

Reply:
Same. I will apply these changes.

Reply:
Done.